karafka.gemspec

```diff
@@ -17,7 +17,7 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'bundler', '~> 1.2'
 
   spec.add_dependency 'ruby-kafka', '= 0.3.17'
-  spec.add_dependency 'sidekiq', '~> 4.2'
+  spec.add_dependency 'sidekiq', '~> 5.0'
```

Comment:
@maximebedard  that is absolutely right. For now, we can open this dependency to support >= 4.2 and everyone should be happy - #148  . Also, I've created #147 to cover that better in the future.